### ISSUE TITLE
slime over tramp just keeps polling

### ISSUE THREAD

**Issue by bon** [slime/slime#478]
I'm trying to run slime over tramp. Here are the steps.
1. `(require 'slime-tramp)`
1. Set `slime-filename-translations`
```emacs-lisp
    (push (list "^lisphost$"
                (lambda (emacs-filename)
                  (subseq emacs-filename (length "/ssh:lisphost.example.com:")))
                (lambda (lisp-filename)
                  (concat "/ssh:lisphost.example.com:" lisp-filename)))
          slime-filename-translations)
```
1. Open a file `/ssh:lisphost.example.com:test.lisp`
1. Give the command `slime`

The result is that it starts lisp on the remote in the buffer `*inferior-lisp*` but gives repeated messages
 ```
Polling "/tmp/slime.29227" .. 157 (Abort with ‘M-x slime-abort-connection’.)
```

I tried setting `temporary-file-directory` to `/ssh:lisphost.example.com:/tmp/` but to no avail.

My slime version is `20181205.944`

**Comment by johnhilts** [comment]
Were you able to find a solution for this? 

I also have this same problem, even when using a tunnel over ssh as per the slime manual: https://slime.common-lisp.dev/doc/html/Setting-up-Emacs.html

**Comment by stassats** [comment]
Are you using the right ports?

**Comment by stassats** [comment]
Well, duh. Slime starts a local instance. You need slime-connect.

**Comment by johnhilts** [comment]
Right, but even when using slime-connect I'm not able to jump to a definition with M-. and I can't compile a file with C-c C-k. 

Any ideas?
